doc/sphinxext/ipython_sphinxext/ipython_directive.py

```diff
@@ -127,7 +127,11 @@
 from sphinx.util.compat import Directive
 
 # Our own
-from IPython import Config, InteractiveShell
+try:
+    from traitlets.config import Config
+except ImportError:
```

Comment:
yeh that seems rather extreme to not to shim. I know *we* get blamed for any teeny tiny little change :)